test/registration/integration_test.rb

```diff
@@ -34,4 +34,16 @@ def test_without_templates
     refute_nil(mod)
     assert_equal('failed', mod['state'], Proxy::LogBuffer::Buffer.instance.info[:failed_modules][:registration])
   end
+
+  def test_chomp_trailing_slash
+    registration_url = 'https://loadbalancer.example.com'
+
+    Proxy::LegacyModuleLoader.any_instance.expects(:load_configuration_file).with('templates.yml').returns(enabled: true, template_url: 'http://smart-proxy.example.com:8000')
+    Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('registration.yml').returns(enabled: true, registration_url: "#{registration_url}/")
+
+    get '/features'
+    response = JSON.parse(last_response.body)
+
+    assert_equal({'registration_url' => registration_url}, response['registration']['settings'])
```

Comment:
Since you're verifying the exposed settings here, can you verify it in `test_with_templates` too?